engine/installation/binaries.md

```diff
@@ -4,243 +4,199 @@ keywords: binaries, installation, docker, documentation, linux
+    ```bash
+    $ sudo docker run hello-world
+    ```
 
-docker/
-docker/docker
-docker/docker-containerd
-docker/docker-containerd-ctr
-docker/docker-containerd-shim
-docker/docker-proxy
-docker/docker-runc
-docker/dockerd
-```
+    This command downloads a test image and runs it in a container. When the
+    container runs, it prints an informational message and exits.
 
-Engine requires these binaries to be installed in your host's `$PATH`.
-For example, to install the binaries in `/usr/bin`:
+### Next steps
 
-```bash
-$ mv docker/* /usr/bin/
-```
+- Continue to [Post-installation steps for Linux](linux-postinstall.md)
 
-> **Note**: Depending on your current setup, you can specify custom paths
-> for some of the binaries provided.
+- Continue with the [User Guide](../../userguide/index.md).
 
-> **Note**: If you already have Engine installed on your host, make sure you
-> stop Engine before installing (`killall docker`), and install the binaries
-> in the same location. You can find the location of the current installation
-> with `dirname $(which docker)`.
+## Install client binaries on macOS
 
-#### Run the Engine daemon on Linux
+The macOS binary includes the Docker client only. It does not include the
+`dockerd` daemon.
 
-You can manually start the Engine in daemon mode using:
+1.  Download the static binary archive. You can download either the latest
+    release binaries or a specific version. To find the download link, see the
+    [release notes](https://github.com/docker/docker/releases) for the version
+    of Docker you want to install. You can choose a `tar.gz` archive or
+    `zip` archive.
 
-```bash
-$ sudo dockerd &
-```
+2.  Extract the archive using `tar` or `unzip`, depending on the format you
+    downloaded. The `docker` binary is extracted.
 
-The GitHub repository provides samples of init-scripts you can use to control
-the daemon through a process manager, such as upstart or systemd. You can find
-these scripts in the <a href="https://github.com/docker/docker/tree/master/contrib/init">
-contrib directory</a>.
+    ```bash
+    $ tar xzvf /path/to/<FILE>.tar.gz
+    ```
 
-For additional information about running the Engine in daemon mode, refer to
-the [daemon command](../reference/commandline/dockerd.md) in the Engine command
-line reference.
+    ```bash
+    $ unzip /path/to/<FILE>.zip
+    ```
+3.  **Optional**: Move the binaries to a directory on your executable path, such
+    as `/usr/local/bin/`. If you skip this step, you must provide the path to the
+    executable when you invoke `docker` or `dockerd` commands.
 
-### Get the macOS binary
+    ```bash
+    $ sudo cp docker/docker /usr/local/bin/
+    ```
 
-The macOS binary is only a client. You cannot use it to run the `docker`
-daemon. To download the latest version for macOS, use the following URLs:
+4.  To connect the `docker` client to a remote host, either set the
+    `DOCKER_HOST` environment variable to the hostname or IP address of the
```

Comment:
I don't think this is necessary - we've just taken reader through local install, they're unlikely to need this